src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -507,7 +507,7 @@ def load_from_zarr(cls, folder, recording=None):
         # sparsity
         if "sparsity_mask" in zarr_root.attrs:
             # sparsity = zarr_root.attrs["sparsity"]
-            sparsity = ChannelSparsity(zarr_root["sparsity_mask"], self.unit_ids, rec_attributes["channel_ids"])
+            sparsity = ChannelSparsity(zarr_root["sparsity_mask"], cls.unit_ids, rec_attributes["channel_ids"])
```

Comment:
Also saw this while I was checking on the zarr stuff too. This should be cls not self.

Reply:
oups. good catch sorry for that.
We should improve tests...
my bad